docs/api.asciidoc

```diff
@@ -328,15 +328,31 @@ custom context and tags.
 [[context-set-tag]]
 ==== `func (*Context) SetTag(key, value string)`
 
-SetTag tags the transaction or error with the given key and value. If the
-key contains any special characters (`.`, `*`, `"`), they will be replaced
-with underscores. Values longer than 1024 characters will be truncated.
-Tags will be indexed in Elasticsearch as keyword fields.
+SetTag is equivalent to calling SetLabel with a string value.
 
-TIP: Before using custom tags, ensure you understand the different types of
+NOTE: This function is deprecated, and will be removed in a future major
```

Comment:
Just as an FYI, we do have fun _deprecated_ tags you can use here as well. As an example:
```
deprecated[4.1.x,Tags have been replaced with labels], Please use <<apm-add-labels,`apm.addLabels()`>> instead
```
You can see what this renders as [here](https://www.elastic.co/guide/en/apm/agent/rum-js/4.x/agent-api.html#apm-add-tags).

Reply:
Thanks, I'll make this change in a follow-up.

Reply:
I don't know that you necessarily have to update this, just wanted to show you in case you didn't know about it.